src/graph.jl

```diff
@@ -148,6 +150,12 @@ function status(sf::StackFrame)
     if !sf.from_c && sf.func === :eval_user_input && endswith(String(sf.file), "REPL.jl")
         st |= repl
     end
+    if !sf.from_c && occursin("./compiler/", String(sf.file))
```

Comment:
Does this work on Windows?

Reply:
Since there are relatively few entry points into compilation, what about being a bit more specific? I worry about, e.g., a symbolic algebra package that may "compile" your expressions but it's a fundamentally different compiler than Julia's.